src/lib/strip-undefined-props.ts

```diff
@@ -0,0 +1,6 @@
+// Hack needed to avoid JSON-Serialization validation error from Next.js https://github.com/zeit/next.js/discussions/11209
+// >>> Reason: `undefined` cannot be serialized as JSON. Please use `null` or omit this value all together.
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function stripUndefinedProperties(obj: any) {
```

Comment:
The API returns null or undefined in the data and we need to strip this before passing to the page component in `getStaticProps`